src/lib/flight_tasks/tasks/AutoMapper/FlightTaskAutoMapper.cpp

```diff
@@ -194,13 +189,11 @@ float FlightTaskAutoMapper::_getLandSpeed()
 		const float land_speed = math::gradual(_dist_to_ground,
 						       _param_mpc_land_alt2.get(), _param_mpc_land_alt1.get(),
 						       _param_mpc_land_speed.get(), _constraints.speed_down);
-		const float head_room = _constraints.speed_down - land_speed;
-
-		speed = land_speed + 2 * (0.5f - throttle) * head_room;
 
-		// Allow minimum assisted land speed to be half of parameter
-		if (speed < land_speed * 0.5f) {
-			speed = land_speed * 0.5f;
+		// user input assisted land speed
+		if (_param_mpc_land_rc_help.get()) {
```

Comment:
Don't modify if the sticks aren't good

```suggestion
		if (_param_mpc_land_rc_help.get() && _sticks.evaluateSticks(_time_stamp_current)) {
```

Reply:
I actually want to do exactly this. I must have postponed it because of the implementation order. We should definitely add it, thanks for pointing out!

Reply:
I added this to the last commit (the rest is unchanged).